### ISSUE TITLE
`Attach as gunmod` do not use colors after #67309

### ISSUE THREAD

**Issue by alef** [CleverRaven/Cataclysm-DDA#67581]
### Describe the bug

See the screenshot. That is what I see in `Attach as gunmod` action from `Use item` menu after `Select gun to modify`.

### Attach save file

N/A

### Steps to reproduce

1. Have a shoulder strap
2. Try attach to a rifle as gunmon
3. Read the text

### Expected behavior

Text at bottom box should be colorized.

### Screenshots

![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/3899581/1a3934ad-2e16-4e2d-b1ee-0359c450f5e0)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.3324 (22H2)
- Game Version: 2553eede89 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth],
    No Rail Stations [No_Rail_Stations]
]

### Additional context

Wrote to #67309 but on Discord suggested to open a new issue.
